pubtools/_quay/signature_handler.py

```diff
@@ -431,7 +432,19 @@ def construct_item_claim_messages(self, push_item):
         claim_messages = []
 
         if push_item.claims_signing_key:
-            digests = self.get_tagged_image_digests(push_item.metadata["pull_url"])
+            if (
```

Comment:
I kind of don't get this actually. Didn't get manifest method simply return manifest in the order ML -> V2S2 -> V2S1? Would that cause issues? What is being fixed here?